spec/support/ems_refresh_helper.rb

```diff
@@ -0,0 +1,45 @@
+module Spec
+  module Support
+    module EmsRefreshHelper
+      def serialize_inventory
+        # These models don't have tables behind them
+        skip_models = [MiqRegionRemote, VmdbDatabaseConnection, VmdbDatabaseLock, VmdbDatabaseSetting]
+        models = ApplicationRecord.subclasses - skip_models
+
+        # Skip attributes that always change between refreshes
+        skip_attrs_global   = ["created_on", "updated_on"]
+        skip_attrs_by_table = {
```

Comment:
It's weird to me that the variable name is `_by_table`, but the keys are model names.

Reply:
Yeah good point, updated to `_by_model`